StellarDotnetSdk.Tests/SorobanServerTest.cs

```diff
@@ -994,6 +1006,117 @@ public async Task TestGetTransactionFails()
         Assert.AreEqual(GetTransactionResponse.GetTransactionStatus.NOT_FOUND, getTransactionResponse.Status);
     }
 
+    [TestMethod]
+    public async Task TestSimulateTransaction()
+    {
+        const string json =
+            "{\n"
+            + "    \"jsonrpc\": \"2.0\",\n"
+            + "    \"id\": \"7a469b9d6ed4444893491be530862ce3\",\n"
+            + "    \"result\": {\n"
+            + "      \"transactionData\": \"AAAAAAAAAAIAAAAGAAAAAem354u9STQWq5b3Ed1j9tOemvL7xV0NPwhn4gXg0AP8AAAAFAAAAAEAAAAH8dTe2OoI0BnhlDbH0fWvXmvprkBvBAgKIcL9busuuMEAAAABAAAABgAAAAHpt+eLvUk0FquW9xHdY/bTnpry+8VdDT8IZ+IF4NAD/AAAABAAAAABAAAAAgAAAA8AAAAHQ291bnRlcgAAAAASAAAAAAAAAABYt8SiyPKXqo89JHEoH9/M7K/kjlZjMT7BjhKnPsqYoQAAAAEAHifGAAAFlAAAAIgAAAAAAAAAAg==\",\n"
+            + "      \"minResourceFee\": \"58181\",\n"
+            + "      \"events\": [\n"
+            + "        \"AAAAAQAAAAAAAAAAAAAAAgAAAAAAAAADAAAADwAAAAdmbl9jYWxsAAAAAA0AAAAg6bfni71JNBarlvcR3WP2056a8vvFXQ0/CGfiBeDQA/wAAAAPAAAACWluY3JlbWVudAAAAAAAABAAAAABAAAAAgAAABIAAAAAAAAAAFi3xKLI8peqjz0kcSgf38zsr+SOVmMxPsGOEqc+ypihAAAAAwAAAAo=\",\n"
+            + "        \"AAAAAQAAAAAAAAAB6bfni71JNBarlvcR3WP2056a8vvFXQ0/CGfiBeDQA/wAAAACAAAAAAAAAAIAAAAPAAAACWZuX3JldHVybgAAAAAAAA8AAAAJaW5jcmVtZW50AAAAAAAAAwAAABQ=\"\n"
+            + "      ],\n"
+            + "      \"results\": [\n"
+            + "        {\n"
+            + "          \"auth\": [\n"
+            + "            \"AAAAAAAAAAAAAAAB6bfni71JNBarlvcR3WP2056a8vvFXQ0/CGfiBeDQA/wAAAAJaW5jcmVtZW50AAAAAAAAAgAAABIAAAAAAAAAAFi3xKLI8peqjz0kcSgf38zsr+SOVmMxPsGOEqc+ypihAAAAAwAAAAoAAAAA\"\n"
+            + "          ],\n"
+            + "          \"xdr\": \"AAAAAwAAABQ=\"\n"
+            + "        }\n"
+            + "      ],\n"
+            + "      \"cost\": { \"cpuInsns\": \"1646885\", \"memBytes\": \"1296481\" },\n"
```

Comment:
Missing the event and cost validations, maybe even more. Can you check it yourself to add the missing validations?

Reply:
I had thought about adding the event validation, then somehow forgot it.
Regarding the cost, it is obsolete, do we really need to validate it?

Reply:
Why is cost obsolete?

Reply:
It's from the official documentation. 
https://developers.stellar.org/network/soroban-rpc/api-reference/methods/simulateTransaction